packages/gatsby-plugin-typography/README.md

```diff
@@ -2,7 +2,7 @@
 
 A Gatsby plugin for utilizing the [Typography](https://kyleamathews.github.io/typography.js/) library with minimal configuration.
 
-See it in action in the [Tutorial](https://www.gatsbyjs.org/tutorial/part-3/)
+See it in action in the [Tutorial](https://www.gatsbyjs.com/tutorial/part-3/)
```

Comment:
This is a dead link

Reply:
This looks like the old url paths for the tutorial; do we not have redirects properly working for these either?

Reply:
It is now `/docs/tutorial` rather than just `/tutorial` but when I looked at this one particularly, the content of the tutorial has changed and `Typography` is no longer a part of it. And we do have a redirect setup for `/tutorial` -> `/docs/tutorial` but not for when theres additional legs to the URL such as "/part-3" here.

I couldn't find the "old" tutorial anywhere — do you know if it still lingering around somewhere?

Reply:
I'm fairly certain the old version has been taken down although the legacy docs sites (i.e. `v3.gatsbyjs.com`) would still have them, but looking at the context of this change, we can probably remove it from this plugin's README

Reply:
Yeah, can be removed